### PR TITLE
ADEN-2468 Add mercury onLoad listener

### PR DESCRIPTION
**Pull request by fraszczakszymon**
We need to know when mercury is fully loaded and has proper ad context set, so here is implementation which starts queue with onLoad callbacks.

### CODE REVIEW

**Comment by jacekjursza** [comment]
lgtm

**Comment by drets** [comment]
App part: https://github.com/Wikia/app/pull/8600

File: front/scripts/mercury/modules/Ads.ts [hunk 1]
```diff
@@ -24,6 +24,9 @@ module Mercury.Modules {
 			get (): number;
 			increment (): number;
 		};
+		private adMercuryListenerModule: {
+			startOnLoadQueue (): void;
```
Comment (bkoval) [inline front/scripts/mercury/modules/Ads.ts]: I believe we don't do space after function  name: https://github.com/Wikia/guidelines/tree/master/TypeScript#object-method-shorthand

File: front/scripts/mercury/modules/Ads.ts [hunk 2]
```diff
@@ -132,6 +138,11 @@ module Mercury.Modules {
 		 */
 		public reloadWhenReady (): void {
 			this.reload(this.currentAdsContext);
+			this.onLoad();
+		}
+
+		private onLoad (): void {
```
Comment (bkoval) [inline front/scripts/mercury/modules/Ads.ts]: I believe we don't do space after function  name: https://github.com/Wikia/guidelines/tree/master/TypeScript#object-method-shorthand

**Pull request merged**